elasticsearch/src/main/resources/META-INF/native-image/io.micronaut.elasticsearch/elasticsearch/reflect-config.json

```diff
@@ -0,0 +1,110 @@
+[
+  {
+    "name": "org.apache.commons.logging.LogFactory"
+  },
```

Comment:
Maybe this is fine for now but I think long term we are going to need a `micronaut-logging` project with `micronaut-log4j` and `micronaut-log4j2` projects that contain this config and can be shared.

Reply:
Agree